docs/rule/no-childless-elements.md

```diff
@@ -0,0 +1,45 @@
+## no-childless-elements
+
+Childless elements
+
+Unexpected childless elements pollute DOM and may lead to rendering bug. This rule forbids childless elements. They have to contain at least one child. Children can be either element nodes or text.
+
+This rule **forbids** the following:
+
+```hbs
+<h1></h1>
+<h2></h2>
+<h3></h3>
+<h4></h4>
+<h5></h5>
+<h6></h6>
+<p></p>
+<span></span>
```

Comment:
the span is an inline element and is often left empty while styled.

Reply:
Added it to the list ! But it lead to having failing tests. The reason is explained here https://github.com/ember-template-lint/ember-template-lint/pull/735#issuecomment-521995961

Reply:
Fixed